src/compas_fab/blender/artists.py

```diff
@@ -13,16 +13,24 @@
     pass
 
 __all__ = [
-    'RobotArtist',
```

Comment:
Shouldn't this stay?

Reply:
It doesn't have to, but it could. The alias is explicitly imported in `blender/__init__.py` and in there, it is exposed in the `__all__`. But I can add it back in if you think it's less confusing.

Reply:
As you wish.